docs/tutorials/shallow-join.rst

```diff
@@ -0,0 +1,359 @@
+Visualizing Analytics in Jupyter
```

Comment:
You need to add an entry to ``docs/tutorials/index.rst`` so that this tutorial is navigable from the main TOC.